dbt_coverage/__init__.py

```diff
@@ -625,8 +645,11 @@ def do_compute(project_dir: Path = Path('.'), cov_report: Path = Path('coverage.
 
     catalog = load_files(project_dir)
     coverage_report = compute_coverage(catalog, cov_type)
-
-    print(coverage_report.to_formatted_string())
+    
+    if str(cov_format) == 'markdown':
```

Comment:
@vvvito same as above -- I believe the [`Enum`](https://typer.tiangolo.com/tutorial/parameter-types/enum/) would make this much nicer to handle 🙂